tests/communication/src/channelmanager.cpp

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright (C) 2023 Renesas Electronics Corporation.
+ * Copyright (C) 2023 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <chrono>
+#include <mutex>
+#include <thread>
+
+#include <zephyr/kernel.h>
+#include <zephyr/tc_util.h>
+#include <zephyr/ztest.h>
+
+#include <aos/common/tools/log.hpp>
+
+#include "communication/channelmanager.hpp"
+#include "utils/checksum.hpp"
+
+#include "stubs/transport.hpp"
+
+static void TestLogCallback(const char* module, aos::LogLevel level, const aos::String& message)
```

Comment:
There is common log callback in tests/utils/log.hpp

Reply:
Done